ivadomed/models.py

```diff
@@ -1430,6 +1430,212 @@ def forward(self, x):
+
+
+class HourglassNet(nn.Module):
+    """Stacked Hourglass Networks.
+       Hourglass network is an auto-encoder decoder architecture for the task of pose estimation. In this architecture, 
+       Features are processed across all scales and consolidated to best capture the various spatial relationships associated with the pose.
+
+    .. seealso::
+        Alejandro Newell et al. "Stacked Hourglass Networks for Human Pose Estimation."
+        Proceedings of the European Conference on Computer Vision. 2016.
+
+    Args:
+        num_stacks (int): number of stacked hourglass networks 
+        num_blocks (int): number of blocks in the encoder structure
+        num_classes (int): number of joints to learn the pose structure
+        name (str): model's name used for call in configuration file.
+    """  
+    def __init__(self, num_stacks=2, num_blocks=4, num_classes=11, block = HGBottleneck, **kwargs):
+        super(HourglassNet, self).__init__()
+
+        self.inplanes = 64
+        self.num_feats = 128
+        self.num_stacks = num_stacks
+        self.conv1 = nn.Conv2d(1, self.inplanes, kernel_size=7, stride=2, padding=3,
+                               bias=True)
+        self.bn1 = nn.BatchNorm2d(self.inplanes)
+        self.relu = nn.ReLU(inplace=True)
+        self.layer1 = self._make_residual(block, self.inplanes, 1)
+        self.layer2 = self._make_residual(block, self.inplanes, 1)
+        self.layer3 = self._make_residual(block, self.num_feats, 1)
+        self.maxpool = nn.MaxPool2d(2, stride=2)
+        self.scale_score = nn.Upsample(scale_factor=4, mode='bilinear')
+
+        # build hourglass modules
+        ch = self.num_feats*block.expansion
+        hg, res, fc, score, fc_, score_ = [], [], [], [], [], []
+        for i in range(num_stacks):
+            hg.append(Hourglass(block, num_blocks, self.num_feats, 4))
+            res.append(self._make_residual(block, self.num_feats, num_blocks))
+            fc.append(self._make_fc(ch, ch))
+            score.append(nn.Conv2d(ch, num_classes, kernel_size=1, bias=True))
+            if i < num_stacks-1:
+                fc_.append(nn.Conv2d(ch, ch, kernel_size=1, bias=True))
+                score_.append(nn.Conv2d(num_classes, ch, kernel_size=1, bias=True))
+        self.hg = nn.ModuleList(hg)
+        self.res = nn.ModuleList(res)
+        self.fc = nn.ModuleList(fc)
+        self.score = nn.ModuleList(score)
+        self.fc_ = nn.ModuleList(fc_)
+        self.score_ = nn.ModuleList(score_)
+
+    def _make_residual(self, block, planes, blocks, stride=1):
+        downsample = None
+        if stride != 1 or self.inplanes != planes * block.expansion:
+            downsample = nn.Sequential(
+                nn.Conv2d(self.inplanes, planes * block.expansion,
+                          kernel_size=1, stride=stride, bias=True),
+            )
+
+        layers = []
+        layers.append(block(self.inplanes, planes, stride, downsample))
```

Comment:
Might be cleaner to just initialize the list right away since we append immediately after initializing the empty list
```suggestion
        layers = [block(self.inplanes, planes, stride, downsample))]
```

Reply:
The initial form is more clear. You have extra `)`.